racket/src/racket/gc2/Makefile.in

```diff
@@ -502,6 +502,11 @@ foreign.@LTO@: $(XSRCDIR)/foreign.c
 main.@LTO@: $(XSRCDIR)/main.c
 	$(CC) $(ALL_CFLAGS) -c $(XSRCDIR)/main.c -o main.@LTO@
 
+gmp_alpha_gcc.@LTO@: $(srcdir)/gmp/gmp_alpha_gcc.s
+	$(AS) -o gmp_alpha_gcc.@LTO@ $(srcdir)/gmp/gmp_alpha_gcc.s
```

Comment:
I'd expect the output path to need a "../src/" prefix, too.

Reply:
Same as above.